src/scripting.md

```diff
@@ -5,66 +5,179 @@ eleventyNavigation:
+<input type="email" id="emailField" class="input input-email">
 ```
-Can be identified by innerText=Delete. The matching is case sensitive and matches the full string.
 
-In order to suppress intermediate steps you need to make sure that data logging is disabled for the steps up to the ones you want to record. For example:
+the following selectors will work:
+
+* ✅ `type=email`
+* ✅ `id=emailField`
+
+and the following selectors will not:
+
+* ❌ `class=input input-email` (spaces inside attribute values are not supported)
+* ❌ `class="input input-email"` (not even if quoted)
+
+Example:
 
 ```markup
-logData    0
+// Set the email to darth@vader.com
+setValue type=email darth@vader.com
+```
 
-// put any urls you want to navigate
-navigate    www.aol.com
-navigate    news.aol.com
+#### `innerText=<text>`
 
-logData    1
+This selector selects elements based on its inner text. For the following button:
 
-// this step will get recorded
-navigate    news.aol.com/world
+```html
+<button type="submit">Login</button>
 ```
 
-The script above will navigate to the main aol portal, then to the news page and then finally to the world-news specific page (logging results for just the world news page). This lets you test the impact of a given path to a site on it's performance (shared css and js caching for example).
+the following selectors will work:
+
+* ✅ `innerText=Login`
+
+and the following selectors will not:
 
-Another significant use case is if you want to test a site that requires authentication. Here is what an authentication script would look like:
+* ❌ `innerText=login` (the selector is case sensitive)
+* ❌ `innerText=log` (the selector must match the full string)
+
+Also, for a different button, this will not work either:
+
+* ❌ `innerText=log in` (spaces inside the text are not supported)
+
+Example:
 
 ```markup
-logData	0
+// Click the Login button
+click innerText=Login
+```
 
-// bring up the login screen
-navigate	http://webmail.aol.com
+#### `innerHtml=<html>`
 
-logData	1
+This selector selects elements based on its inner HTML. For the following link:
 
-// log in
-setValue	name=loginId	someuser@aol.com
-setValue	name=password	somepassword
-submitForm	name=AOLLoginForm
+```html
+<a href="/dashboard"><span>Dashboard</span></button>
 ```
 
-You won't get a lot of feedback as to why a script failed. For debugging purposes it is easiest to limit scripts to navigate and exec/execAndWait commands which can be debugged locally in a browser's dev tools.
+the following selectors will work:
+
+* ✅ `innerHtml=<span>Dashboard</span>`
+
+and the following selectors will not:
+
+* ❌ `innerHtml=<span>dashboard</span>` (the selector is case sensitive)
+* ❌ `innerHtml=span` (you can’t select just on tags)
+
+Also, for a different link, this will not work either:
+
+* ❌ `innerText=<span>Open Dashboard</span>` (spaces inside the HTML are not supported)
+
+Example:
+
+```markup
+// Click the Dashboard link
+click innerHtml=<span>Dashboard</span>
```

Comment:
Does the example look correct? I never used this selector (and my tests in WPT don’t seem to be working).